content/blog/2018-00-00-havanar.md

```diff
@@ -9,29 +9,30 @@ title: HavanaR Workshop 2018
 slug: havanar
 ---
 
-The first meeting of R users in Cuba, HavanaR! club 2018, will take place From 29-10-2018 to 02-11-2018. The workshop is organized jointly by the Group of Probability and Statistics of Universidad de La Habana, the >eR-Biostat initiavle, I-Bioest, Hasselt University, Belgium and the R Forward taskforce.
+The first meeting of R users in Cuba, HavanaR! Club 2018, will take place From 29-10-2018 to 02-11-2018. The workshop is organized jointly by the Group of Probability and Statistics of Universidad de La Habana, the >eR-Biostat initiative, the Center for Statistics, CenStat & I-Biostat, Hasselt University, Belgium and the R Forward taskforce.
```

Comment:
On our own website the taskforce can be referred to simply as "Forwards". (This is the official name, but often the R is added to clarify what we're about, especially in a general environment such as Twitter, vs a specific R environment, such as useR!).